src/rnanorm/command.py

```diff
@@ -0,0 +1,78 @@
+"""Command-line commands."""
+import argparse
+import os
+
+import pandas as pd
+
+from .normalization import tpm
+
+
+def main():
```

Comment:
Is there a reason why this function is not in the `__main__,py` file? I would suggest using a more meaningful name if you keep it here.

Reply:
@acopar told me he tried to include `main()` in  `__main__.py`, but could not get it to work. Maybe Flit can not have entry points in modules that start with `__`?

A meaningful name would be `rnanorm`. But then it's the same as the package name... :)

Reply:
This was a temporary solution because in previous versions the following configuration failed to link properly. It can be placed in  `__main__.py` (tested without wrapper function) and used with the following pyproject entrypoint: `rnanorm:__main__`.

Reply:
> tested without wrapper function

Reply:
> tested without wrapper function

I wanted a wrapper because I intend to write tests someday.

Reply:
@acopar Would you move the `main()` method to `__name__` after this PR is merged and test if it links properly? 

What is the community standard for testing commands with unit tests? Would you write a simple unit test? If you need example data, use the example in `test_tpm_normalization`. You can create a mini expressions file and a mini gene lengths file.